### DISCUSSION TITLE
What software do you use to create these icons?

### DISCUSSION THREAD

**Discussion by Robinerra** [tabler/tabler-icons#252] [General]
Looking to help contribute since there's a backlog of requests but I can't seem to find what software is being used to make these icons, nor can I find a contribution guide. Thanks.

**Comment by codecalm** [comment]
i'm using Sketch at this moment, and export icons to repository by `_import.sh` script :) 

<img width="726" alt="Screenshot 2022-07-23 at 13 22 31" src="https://user-images.githubusercontent.com/1282324/180602961-682f14aa-c5e1-4d8c-9e94-2a244fb0aafd.png">
